Proton/Sources/Attachment/Attachment.swift

```diff
@@ -356,9 +356,9 @@ open class Attachment: NSTextAttachment, BoundsObserving {
 
     func render(in editorView: EditorView) {
         self.containerEditorView = editorView
-        guard view.superview == nil else { return }
+        guard !isRendered else { return }
         editorView.richTextView.addSubview(self.view)
-        self.view.layoutIfNeeded()
+        self.view.layoutIfNeeded() // this triggers re-layout of all attachments, and some times the whole UITableView
```

Comment:
Leaving a comment here as a reminder of how problematic this may be for some view hierarchies.